bin/api.sh

```diff
@@ -79,6 +79,10 @@ if [[ -n "$RPC_HEALTH_CHECK_SLOT_DISTANCE" ]]; then
   args+=(--health-check-slot-distance "$RPC_HEALTH_CHECK_SLOT_DISTANCE")
 fi
 
+if [[ -n "$ACCOUNT_INDEXES" ]]; then
```

Comment:
Multiple `--account-index` args are supported right?  If so try following the `TRUSTED_VALIDATOR_PUBKEYS` pattern instead

Reply:
Sure, I can do that. Is there a reason why a string like "program-id spl-token-mint spl-token-owner" won't work, though?

Reply:
lol oh does it?   I assumed it wouldn't.   At the very least you'd run into shell escaping issues at some point, such as `--account-index program-id spl-token-mint spl-token-owner` vs `--account-index "program-id spl-token-mint spl-token-owner"`

Reply:
Yup, it does work from the command line and in the local env-variable test I ran. But I've no problem being verbose/explicit to avoid potential shell-escaping wrinkles.